### PR TITLE
8431 Added headers for fetching fees

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#8431

*Description of changes:*
- added headers to getPaymentFees() data
- app version = 2.3.8

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

**Review by eve-git — APPROVED** [review]
LGTM

**Review by vysakh-menon-aot — APPROVED** [review]

File: src/mixins/payment-mixin.ts [hunk 1]
```diff
@@ -129,11 +132,11 @@ export class PaymentMixin extends Mixins(ActionMixin) {
...
-    const data = {
-      // Comment this out to use direct pay
-      /* paymentInfo: {
-        methodOfPayment: methodOfPayment
-      }, */
+    const data: any = {
+      // Comment this out to use direct pay:
+      // paymentInfo: {
+      //   methodOfPayment: methodOfPayment
+      // },
```
Comment (cameron-freshworks) [inline src/mixins/payment-mixin.ts:139]: Is this for local testing or meant to be left in?

**Review by cameron-freshworks — COMMENTED** [review]
Looks good, just had 1 question!

File: src/mixins/payment-mixin.ts (hunk 1 above)
Comment (severinbeauvais) [reply src/mixins/payment-mixin.ts:139]: This was there before. I only changed the comment style.
I'm not afraid to delete obsolete code, but this code looks like we might use it some day.

Comment (cameron-freshworks) [reply src/mixins/payment-mixin.ts:139]: Sounds good

**Review by cameron-freshworks — APPROVED** [review]

**Pull request merged**